pyproject.toml

```diff
@@ -61,23 +61,17 @@ lint.ignore = [
   "B028",    # No explicit stacklevel keyword argument found
   "B904",    # Within an except clause, raise exceptions with ...
   "C408",    # unnecessary-collection-call
-  "COM812",  # missing trailing comma
   "D105",    # Missing docstring in magic method
   "D205",    # 1 blank line required between summary line and description
   "D212",    # Multi-line docstring summary should start at the first line
-  "ISC001",
-  "NPY002",  # replace legacy numpy.random with numpy.random.Generator
   "PD901",   # pandas-df-variable-name
   "PERF203", # try-except-in-loop
   "PERF401", # manual-list-comprehension (TODO fix these or wait for autofix)
```

Comment:
I forgot to remove this `TODO` tag, would do it in another PR.

I'm pretty sure we don't really want to enable this rule:
> it seems to force all kind of list builder into list comprehensions (especially some nested conditions list), which would greatly reduce readability.